README.md

```diff
@@ -1,9 +1,9 @@
 # Carfield
```

Comment:
The readme went though major changes due to automatic scripts of mkdocs, the removed lines are not lost but are now in the docs